src/schemas/utilities/definitions.ts

```diff
@@ -0,0 +1,8 @@
+import { Schema, SchemaDefinition, SchemaProperty } from '@/schemas/types/schema'
+
+export function getSchemaDefinition(schema: Schema, definition: SchemaDefinition): SchemaProperty {
+  const definitionKey = definition.replace('#/definitions/', '')
+  const definitionSchema = schema.definitions?.[definitionKey]
+
+  return definitionSchema ?? {}
```

Comment:
Under what circumstances would there be no definition? I'd almost rather this throw an error that we can handle in the component instead of passing back what I have to assume is an invalid definition (though maybe not strictly invalid from a TS perspective)

Reply:
Agreed, I'm fine with just throwing an error here rather than returning an empty definition.

Reply:
Updated